### DISCUSSION TITLE
Is there anyway to seed multi files into torrent

### DISCUSSION THREAD

**Discussion by Ns2Kracy** [anacrolix/torrent#967] [General]
Is there anyway to seed multi files into torrent, or symlink them to a dir then seed the dir?

**Comment by anacrolix** [comment]
Yes. There might be a demo of this floating around. I think it was called serve or something.

**Reply by Ns2Kracy** [reply]
I am trying to modify the `BuildFromPath` and other else to reach this, then I can add multi files like `info.BuildFromPath(path1, path2, path3)`.
